.github/ISSUE_TEMPLATE/release_new_version.md

```diff
@@ -0,0 +1,70 @@
+---
+name: 'For maintainers: Release a new version'
+about: This is the Developer-only template which we use for tracking the process of future releases.
+title: 'Prepare Release 3.y.z'
+labels: release
+assignees: ''
+---
+
+**Target timeline**
+Schuled feature freeze / Start of translation process: <!-- Add a date, usually about 6-8 weeks from the last release, e.g. 2021-04-20 -->
+Approximate release date: <!-- Usually 2-3 weeks after starting translations, e.g. 2021-05-04 -->
+
+**Checklist**
+- [ ] Assign this issue to the release shepherd who is in charge of managing this checklist
+- [ ] Create a milestone and add all relevant issues
+- [ ] Ensure that all issues/PR targeted for this release are done by checking the milestone
+- [ ] Declare a code freeze. PRs can still be worked on and may get reviewed, but must not be merged unless agreed explicitly.
+- [ ] Write the Changelog
+- [ ] [Tag a beta release](https://github.com/jamulussoftware/jamulus/blob/master/RELEASE-PROCESS.md#steps-for-a-specific-release)
+- [ ] Announce the beta release on Github Discussions
+- [ ] Get feedback on the stability of the beta release
+- [ ] Website translations
+  - [ ] Create new branch `translate3_y_z` based on branch `release`
+  - [ ] Squash-merge `changes` into `translate3_y_z`
+  - [ ] Ping translators. Translators should open PRs with their changes against the `translate3_y_z` branch (?) and reference this issue ("Add German translations for 3.y.z")
+  - [ ] Wait for all PRs to be merged
```

Comment:
Should we agree what happens if the PRs don't come in by the target date? For the web site, do we delete any affected .md files in the wiki/[lang]/ directory so that out-of-date translations don't exist? The language switcher link will still be there but just kicks back to the English. Not sure what to do for the app.

Reply:
> Not sure what to do for the app.

Nothing. :)
Untranslated strings will simply appear in English and I think that's fine. If late translations come in, we could do a very quick .1 release.

I have no strong opinion on the website stuff. If the outdated translation becomes wrong (i.e. because it lacks a required compatibility update -- unlikely, I guess?) we should probably back it out. If it is simply missing some documentation about new features, it might be OK to keep it? Would be nice to have a banner like "This is outdated, see the English version for the latest updates"? Probably not trivial though. :(

Reply:
Yes it's hard to know what to do. People have a right to expect up to date translations or none at all really, since a message that what they're reading is out of date is almost worse! We did in fact take a decision to only translate the "core" pages for now. But that seems to have been sort of ignored though...